objects/src/accounts/code.rs

```diff
@@ -168,8 +169,9 @@ impl Serializable for AccountCode {
 
 impl Deserializable for AccountCode {
     fn read_from<R: ByteReader>(source: &mut R) -> Result<Self, DeserializationError> {
-        let mut module = ModuleAst::read_from(source, MODULE_SERDE_OPTIONS)?;
-        module.load_source_locations(source)?;
+        let module = ModuleAst::read_from(source, MODULE_SERDE_OPTIONS)?;
+        // FIXME: this is failing even when serializing and de-serializing a proven mint tx
+        // module.load_source_locations(source)?;
```

Comment:
Similar comment to the one above.

Reply:
done! Also did the same for the tests at `accounts::data::tests`